### ISSUE TITLE
(2023.05)  PCUI release broken in us-gov-west-1

### ISSUE THREAD

**Issue by tmscarla** [aws/aws-parallelcluster-ui#236]
### Bug description

Due to an issue with ParallelCluster UI **2023.05**, deployment of PCUI in GovCloud west region will fail with the following error message show in the CloudFormation events :

```
Partition "aws" is not valid for resource "arn:aws:ce:us-east-1::/*".
 (Service: AmazonIdentityManagement; Status Code: 400; Error Code: MalformedPolicyDocument; Request ID: 3ef760f7-3467-487a-8eb3-130712e1791c; Proxy: null)
```

The issue is rooted in the fact that the Cost monitoring feature is not supported in GovCloud west region and trying to activate it will result in an error. (See https://github.com/aws/aws-parallelcluster-ui/issues/213)

### Mitigation

You can find a detailed explanation and the mitigation of the problem [here](https://github.com/aws/aws-parallelcluster-ui/wiki/(2023.05)--PCUI-release-broken-in-us-gov-west-1).

**Comment by lazyoft** [comment]
Reopening until the official release.

**Comment by BarcoMasile** [comment]
Closing this issue since it's solved after PCUI 2023.06.0 [release](https://github.com/aws/aws-parallelcluster-ui/releases/tag/2023.06.0).
